flink-runtime/src/main/java/org/apache/flink/runtime/clusterframework/BootstrapTools.java

```diff
@@ -659,10 +659,13 @@ public static String getDynamicPropertiesAsString(Configuration baseConfig, Conf
 	 * For unix-like OS(Linux, MacOS, FREE_BSD, etc.), each value will be surrounded with single quotes. This works for
 	 * all chars except single quote itself. To escape the single quote, close the quoting before it, insert the escaped
 	 * single quote, and then re-open the quoting. For example, the value is foo'bar and the escaped value is
-	 * 'foo'\''bar'.
+	 * 'foo'\''bar'. Navigate to {@see https://en.wikibooks.org/wiki/Windows_Batch_Scripting} for more information about
+	 * windows escape.
 	 * For windows OS, each value will be surrounded with double quotes. The double quote itself needs to be escaped with
 	 * back slash. Also the caret symbol need to be escaped with double carets since it is the window command uses it to
-	 * escape characters.
+	 * escape characters. Navigate to
+	 * {@see https://stackoverflow.com/questions/15783701/which-characters-need-to-be-escaped-when-using-bash} for more
+	 * information about unix escape.
```

Comment:
I guess you meant to add the references the other way around.

Reply:
Exactly. Thanks for your kindly update.